### PR TITLE
Only show pokémons available in the game

### PR DESCRIPTION
**Pull request by EdenGottlieb**
Hides pokemon that are not available in the game from notify and exclude lists.
## Description

Currently, the notify and exclude lists show all existing pokemons from all gens.
This PR shows only pokemon currently available in the game, plus the legendary ones and ditto that will be added later (all pokemons up to #151). This declutters both the exclude and notify boxes and makes searching for pokemons easier. This is done by limiting the key to 151 while transferring JSON data to the search boxes, in a manner that does not damage the original json lists for future use.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by brycedorn** [comment]
good consideration, but rather than continuing to iterate through all of `data` (even after the 151st is reached) you can do `if (key > 151){ return false; }` 

also indentation's a little off.

**Comment by EdenGottlieb** [comment]
@brycedorn Switched to your solution, seriously more efficient. I actually tried it yesterday while working on this but for some reason it didn't seem to work, I was simply having mad JS problems...

Also fixed indentation, My computer and GitHub don't seem to interpret spaces and dashes in the same manner

File: static/map.js [hunk 1]
```diff
@@ -1052,6 +1052,8 @@ $(function () {
         var pokeList = []
 
         $.each(data, function(key, value) {
+            if (key<=151)
+            	return false;
```
Comment (brycedorn) [inline static/map.js]: super minor, but should add brackets around this `if` block to follow convention of the rest of this file

**Comment by brycedorn** [comment]
one last comment then looks 👌!

**Comment by EdenGottlieb** [comment]
@brycedorn Done

**Comment by przerwap** [comment]
If we're to follow convention here, there should be a space before and after the `<=`

**Comment by EdenGottlieb** [comment]
@przerwap Done

File: static/map.js [hunk 2]
```diff
@@ -1052,6 +1052,9 @@ $(function () {
         var pokeList = []
 
         $.each(data, function(key, value) {
+            if (key <= 151) {
```
Comment (AHAAAAAAA) [inline static/map.js]: return false if it IS first gen? conditional looks wrong.

**Comment by niklaswa** [comment]
Hey, it was already included in #1948 
look here: https://github.com/AHAAAAAAA/PokemonGo-Map/blob/develop/static/map.js#L1107

File: static/map.js (hunk 2 above)
Comment (EdenGottlieb) [inline static/map.js]: Damn. Fixed.

**Pull request closed without merging**